### DISCUSSION TITLE
problems with train embeddings and generate images in Automatic1111 Colab

### DISCUSSION THREAD

**Discussion by Idamarinella** [TheLastBen/fast-stable-diffusion#1252] [Q&A]
hi, I have some problems:
 - the first problem is  with the Train in Automatic 1111, when I try to preprocess the images it give me an error, it does not find the input folders for images to be processed in google drive. Where should I put the input/output folders in my google drive to process the images to be towed for embeddings?
- the second one is with the number of batch to generate in text to image and Img2Img too, if I want to generate three or more batch, it start but when it finished it dosn'T show any images and don't restart when i try a new generation, I must reload the UI.
I'm usually copy the colab in my google drive account.
Thank You

**Comment by TheLastBen** [comment]
Hi, to keep the colab updated, don't copy it to your gdrive, or at least update it from time to time with the latest one.

to get the path to your dataset, it's best to use the colab file explorer to copy the path, then paste it in the ui.

to avoid the freezing problem, check the box gradio_server, which is checked by default in the recent colabs.

**Comment by Idamarinella** [comment]
Thank you!

Inviato da Posta<https://go.microsoft.com/fwlink/?LinkId=550986> per Windows

Da: ***@***.***>
Inviato: mercoledì 4 gennaio 2023 15:38
A: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Oggetto: Re: [TheLastBen/fast-stable-diffusion] problems with train embeddings and generate images in Automatic1111 Colab (Discussion #1252)


Hi, to keep the colab updated, don't copy it to your gdrive, or at least update it from time to time with the latest one.

to get the path to your dataset, it's best to use the colab file explorer to copy the path, then paste it in the ui.

to avoid the freezing problem, check the box gradio_server, which is checked by default in the recent colabs.
